docs/content/guides/developer/nft/nft-soulbound.mdx

```diff
@@ -0,0 +1,38 @@
+---
+
+A soulbound NFT is an NFT that is non-transferable. Once a NFT is minted to a Sui account, the NFT is "bounded" to that account and cannot be transferred. This implementation leverages the custom logic of the Sui framework's transfer functions. The [`sui::transfer` module](/references/framework/sui-framework/transfer.md) contains two functions that transfers objects: [`transfer::transfer`](/references/framework/sui-framework/transfer.md#function-transfer) and [`transfer::public_transfer`](/references/framework/sui-framework/transfer.md#function-public_transfer). 
+
+Typically, when defining new NFTs or object types on Sui, you don't need to create a transfer function because the Sui Framework offers `transfer::public_transfer` which anyone can use to transfer objects. However, `transfer::public_transfer` requires transferred objects have the `key` and `store` ability. Therefore, if you define a new NFT type that has the `key` ability, meaning it is a Sui object, but not the `store` ability, the holders won't be able to use `transfer::public_transfer`. This results in a soulbound NFT.
+
+It is also possible to create custom transfer logic for NFTs on Sui. The `transfer::transfer` function has custom rules performed by the Sui Move bytecode verifier that ensures that the transferred objects is an object defined in the module where transfer is invoked. While removing the `store` ability from a struct definition makes `transfer::public_transfer` unusable, `transfer::transfer` can still be used as long as it's used in the module that defined that object's type. This allows for the module owner to provide custom transfer logic for their soulbound NFTs. 
+
+## Example
+
+The following example creates a basic soulbound NFT on Sui. The `TestnetSoulboundNFT` struct defines the NFT with an `id`, `name`, `description`, and `url` fields. 
+
+{@inject: examples/move/nft-soulbound/sources/testnet_soulbound_nft.move#struct=TestnetSoulboundNFT noComments}
+
+This `TestnetSoulboundNFT` struct is defined with the `key` ability but without the `store` ability meaning it cannot be transferred with `transfer::public_transfer` and can only be transferred with `transfer::transfer`, which requires custom transfer logic to be implemented in the same module. 
+
+This example also includes an example of how to provide custom transfer logic using the `transfer::transfer` function. This is where you can add additional logic such as reset the NFT's stats, or requiring a payment. Don't provide this functionality if the NFT is fully soulbound. 
+
+{@inject: examples/move/nft-soulbound/sources/testnet_soulbound_nft.move#fun=transfer}
+
+<details>
+<summary>
+`testnet_soulbound_nft.move` 
+</summary>
+
+{@inject: examples/move/nft-soulbound/sources/testnet_soulbound_nft.move}
+
+</details>
+
+## Related links
+
+- [Soulbound NFT Example source code](https://github.com/MystenLabs/sui/tree/main/examples/move/nft-soulbound): The source code that this document references.
```

Comment:
```suggestion
- [Soulbound NFT example source code](https://github.com/MystenLabs/sui/tree/main/examples/move/nft-soulbound): The source code that this document references.
```
Nittiest of nits. I see why you formatted it the way you did, but I think this is ever so slightly more correct.